tests/FSharp.Compiler.ComponentTests/ErrorMessages/InvalidNumericLiteralTests.fs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
 
-namespace FSharp.Compiler.ComponentTests
+namespace FSharp.Compiler.ErrorMessages.ComponentTests
 
 open Xunit
 open FSharp.Test.Utilities
 open FSharp.Compiler.SourceCodeServices
 
-module ``Invalid Numeric Literal`` =
+module ``Numeric Literlas`` =
```

Comment:
```suggestion
module ``Numeric Literals`` =
```

Reply:
Thanks, I probably need to set up a spellchecker 🤦